frontend/components/site/SiteScans.jsx

```diff
@@ -0,0 +1,265 @@
+import prettyBytes from 'pretty-bytes';
+import { sandboxMsg } from '../../util';
+import {
+  dateAndTimeSimple,
+  timeFrom,
+  dateAndTime,
+  dateOnly,
+} from '../../util/datetime';
+
+import LoadingIndicator from '../LoadingIndicator';
+import GithubBuildBranchLink from '../GithubBuildBranchLink';
+import GithubBuildShaLink from '../GithubBuildShaLink';
+import AlertBanner from '../alertBanner';
+
+import { useSiteBuildTasks } from '../../hooks/useSiteBuildTasks';
+import { useBuildTasksForSite } from '../../hooks/useBuildTasksForSite';
+import { currentSite } from '../../selectors/site';
+import { getOrgById } from '../../selectors/organization';
+import ScanResultsSummary from '../ScanResultsSummary';
+import FilterIndicator from '../FilterIndicator';
+
+const {
+  setDate, isBefore, startOfToday, addMonths,
+} = require('date-fns');
+
+function SiteScans() {
+  const { id } = useParams();
+  const site = useSelector(state => currentSite(state.sites, id));
+  const organization = useSelector(state => getOrgById(state.organizations, site.organizationId));
+  const { buildTasks: scans, isLoading } = useBuildTasksForSite(id);
+  const { siteBuildTasks } = useSiteBuildTasks(id);
+  const [searchParams, setSearchParams] = useSearchParams(false);
+  const today = startOfToday();
+
+  function nextScanDate(siteBuildTask) {
+    if (!siteBuildTask.metadata.runDay) return null;
+    const thisMonthScan = setDate(today, siteBuildTask.metadata.runDay);
+    if (isBefore(thisMonthScan, today)) {
+      return addMonths(thisMonthScan, 1);
+    }
+    return thisMonthScan;
+  }
+
+  if (!site || !scans) return null;
+  const branchToBeScanned = site?.defaultBranch || 'most recently built';
+  const buildIdToFilterBy = Number(searchParams.get('build'));
+  // if filter isn't set or is not set to a valid build ID (positive int), show all
+  /* eslint-disable-next-line max-len */
+  const filteredScans = scans?.filter(scan => !buildIdToFilterBy || scan.buildId === buildIdToFilterBy);
+  function clearParams() { setSearchParams({}); }
+
+  return (
+    <div>
+      <div>
+        { organization?.isSandbox
+          && (
+          <AlertBanner
+            status="warning"
+            message={sandboxMsg(organization.daysUntilSandboxCleaning, 'site scans')}
+            alertRole={false}
+          />
+          )}
+      </div>
+      { isLoading
+        ? <LoadingIndicator />
+        : (
+          <>
+            <div>
+              <h3>Welcome to the Site Scans beta!</h3>
+              <p>
+                {/* eslint-disable-next-line max-len */}
+                Pages is now offering automated monthly quality scans of customer production sites. These scans examine your Pages site for common website issues and provide guidance and resources for remediation. Site Scans are part of a larger effort to help our customers comply with OMB Memo 23-22 and ATO requirements.
+              </p>
+              { siteBuildTasks.length > 0 && (
```

Comment:
We can have site build tasks related to a site but without a set scheduled date. We could use the `nextScanDate(task)` function to filter out this scheduled build tasks table. 

Also, should we not show scheduled builds if we are filtering the scans by Build Id? It can be a bit confusing when you click on build's scan and you see the scheduled scans.